crates/matrix-sdk/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 - `Common::members` and `Common::members_no_sync` take a `RoomMemberships` to be able to filter the
   results by any membership state.
+  - `Common::active_members(_no_sync)` and `Common::joined_members(_no_sync)` are deprecated.
```

Comment:
Same thing here. Though I don't know who uses the APIs you've deprecated, if they're not that likely to be used maybe it's okay? That also makes me wonder if we should keep `room.members()` as a function without extra arguments and name the new function something like `members_filtered()` 🤔

Reply:
Yay `Deref` 😶

Reply:
We could remove the `no_sync` methods from `Common` and change the names in `BaseRoom` to `no_sync`.

Reply:
If that is equivalent in functionality, seems like a good solution given the constraints (but at some point I really want to get rid of those `Deref`s)

Reply:
Oh actually I'm wrong. It"s not the same. I got confused by the type names that are the same in both crates.